config/dbconfig.txt

```diff
@@ -29,6 +29,11 @@ FEEDBACK_LOGIN username
 ## Password used to access the database.
 FEEDBACK_PASSWORD password
 
-## Time in deciseconds for a query to execute before alerting a for possible slow query timeout.
-## While enabled queries and their execution times are logged if they exceed this value.
-#QUERY_DEBUG_LOG_TIMEOUT 70
+## Time in seconds for asynchronous queries to timeout
+## Set to 0 for infinite
+ASYNC_QUERY_TIMEOUT 10
+
+## Time in seconds for blocking queries to execute before alerting a for possible slow query timeout 
+## Set to 0 for infinite
+## Must be less than or equal to ASYNC_QUERY_TIMEOUT
+BLOCKING_QUERY_TIMEOUT 5
```

Comment:
@MrStonedOne

Reply:
If the timeout elapses it runs the current slow query handler

Reply:
Re read it again

Reply:
word _for_ word

Reply:
communicate clearly

Reply:
read it word _for_ word

or rather

read it _for_ word word.

Reply:
>Time in seconds for blocking queries to execute before alerting a for possible slow query timeout 

Read it slowly.

Reply:
you can just say, there's an extra a before your for in this sentence

you don't have to play 20 questions, we're not on the chaser

Reply:
but there isn't.

Reply:
the isn't an extra `a`.

Reply:
#39173

Reply:
there is an extra a

Reply:
no, there is exactly the right amount of a's in that sentence, its just that one of them appears 1 word too soon.